QRCoder/PayloadGenerator.cs

```diff
@@ -2536,7 +2532,7 @@ public byte[] ToBytes()
             {
                 //Setup byte encoder
                 //Encode return string as byte[] with correct CharacterSet
-#if !NET35_OR_GREATER
+#if !NETFRAMEWORK   // -- why different ifdef than line 2519? --
```

Comment:
> -- why different ifdef than line 2519? --

No clue, but I would say the latter is the correct one: https://nicolaiarocci.com/how-to-read-windows-1252-encoded-files-with-.netcore-and-.net5-/

Reply:
Updated